CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Blip is the web front end for YourLoops system.
 It is based on Tidepool Blip 1.27.
 
 ## 3.0.5-beta - 2023-07-12
+### Engineering use
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Is it really an Engineering use thing? This is not a refactor or something that the patient won't see. It looks like a feature, don't you think?

Reply:
I had a doubt about that, ok for a feature ;)